core/codenvy-hosted-api-workspace/src/main/java/com/codenvy/api/workspace/server/filters/WorkspacePermissionsFilter.java

```diff
@@ -79,7 +79,6 @@ public void filter(GenericResourceMethod genericResourceMethod, Object[] argumen
                 workspaceId = ((String)arguments[0]);
                 action = DELETE;
                 break;
-            case "recoverWorkspace":
```

Comment:
I think this is pretty similar to interceptors compatibility tests like [this one](https://github.com/codenvy/customer-saas/blob/98fdeece736cebf6da844f3645b44cad1bbfa2b8/wsmaster/saas-marketo/src/test/java/com/codenvy/saas/marketo/UpdateProfileInterceptorTest.java). The main point here is that if intercepted class doesn't contain such method build MUST fail.